ibm/resource_ibm_appid_token_config.go

```diff
@@ -78,7 +78,7 @@ func resourceIBMAppIDTokenConfig() *schema.Resource {
 						"destination_claim": {
 							Description: "Optional: Defines the custom attribute that can override the current claim in token.",
 							Type:        schema.TypeString,
-							Required:    true,
+							Optional:    true,
```

Comment:
@dariusbakunas, Please update docs accordingly

Reply:
It was already optional in docs